tests/e2e/testutils.go

```diff
@@ -206,3 +206,22 @@ func RunCommand(cmd string) (string, error) {
 	}
 	return out.String(), nil
 }
+
+func UpgradeCluster(serverNodenames []string, agentNodenames []string) error {
+	for _, nodeName := range serverNodenames {
+		cmd := "RELEASE_CHANNEL=commit vagrant provision " + nodeName
```

Comment:
Can we make this a const above the loops and reference that in this loop and the one below?